enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -0,0 +1,356 @@
+or by modifying certificates present in their [trustedCA ConfigMap](https://docs.openshift.com/container-platform/4.12/security/certificates/updating-ca-bundle.html#ca-bundle-replacing_updating-ca-bundle).
+
+In all cases, Windows nodes can be joined to the cluster after altering proxy settings, which would result in WMCO 
+applying proxy settings during initial node configuration. In the latter 2 scenarios, Windows nodes may already exist in
+the cluster, in which case WMCO would reconcile to update them.
+
+### Risks, Drawbacks, and Mitigations
+
+The risks and mitigations are similar to those on the [Linux side of the cluster-wide proxy](https://github.com/openshift/enhancements/blob/master/enhancements/proxy/global-cluster-egress-proxy.md#risks-and-mitigations).
+Although cluster infra resources already do a best effort validation on the user-provided proxy URL schema and CAs,
+a user could provide non-functional proxy settings/certs. This would be propogated to their Windows nodes and workloads,
+taking down existing application connectivity and preventing new Windows nodes from being bootstrapped.
+
+The only drawbacks are the increased complexity of WMCO and the potential complexity of debugging customer cases that
+involve a proxy setup, since it would be extremely difficult to set up an accurate replicatation environment.
+This can be mitigated by proactively getting the development team, QE, and support folks familiar with the expected
+behavior of Windows nodes/workloads in proxied clusters, and comfortable spinning up their own proxied clusters.
+
+### API Extensions
+
+N/A, as no CRDs, admission and conversion webhooks, aggregated API servers, or finalizers will be added or modified.
+Only the WMCO will be extended which is an optional operator with its own lifecycle and SLO/SLAs, a 
+[tier 3 OpenShift API](https://docs.openshift.com/container-platform/4.12/rest_api/understanding-api-support-tiers.html#api-tiers_understanding-api-tiers).
+
+### Operational Aspects of API Extensions
+
+N/A
+
+#### Failure Modes & Support Procedures
+
+In general the support procedures for WMCO will remain the same. There are two underlying mechanisms we rely on, the
+publishing of proxy config to cluster resources and the comsuming of the published config. If either of the underlying
+mechanisms fail, Windows nodes will become proxy unaware. This could involve an issue with the user-provided proxy
+settings, the cluster network operator, OLM, or WMCO. This would result in all future egress traffic circumventing the
+proxy, which could affect inter-pod communication, existing application availablity, and security. Also, the pause image
+may not be able to be fetched, preventing new Windows nodes from being bootstrapped. This would require manual
```

Comment:
Is the pause image pulled required during bootstrapping? I thought it happens when the first Pod is deployed.

Reply:
You're right. I will edit this